content/manuals/testcontainers.md

```diff
@@ -42,6 +42,8 @@ Testcontainers provide support for the most popular languages, and Docker sponso
 
 The rest are community-driven and maintained by independent contributors.
 
+{{< embedded language="go" source="/git-src/testcontainers-go/modules/redis/examples_test.go" id="runRedisContainer" >}}
```

Comment:
Before merging this PR, this line is just an example on how to use it, so it should be removed from here.